packages/qdrant-loader/src/qdrant_loader/connectors/localfile/connector.py

```
@@ -80,126 +82,184 @@ def set_file_conversion_config(self, file_conversion_config: FileConversionConfi
+
+        # Add file conversion metadata if applicable
+        if needs_conversion:
+            metadata.update(
+                {
+                    "conversion_method": conversion_method,
+                    "conversion_failed": conversion_failed,
+                    "original_file_type": original_file_type,
+                }
+            )
+
+        self.logger.debug(f"Processed local file: {rel_path.replace('\\', '/')}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)